[AWS] Update catalog from host periodically (#1451)
skypilot-org/skypilot: docs/source/reference/faq.rst

(Advanced) How to make SkyPilot use all global regions?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> By default, SkyPilot only supports the US regions on different clouds (except for AWS) for convenience. If you want to utilize all global regions, please run the following command:

Reviewer: ```suggestion
By default, SkyPilot supports most global regions on AWS and only supports the US regions on GCP and Azure. If you want to utilize all global regions, please run the following command:
```

.. code-block:: bash

  version=$(python -c 'import sky; print(sky.clouds.service_catalog.constants.CATALOG_SCHEMA_VERSION)')
  mkdir -p ~/.sky/catalogs/${version}
  cd ~/.sky/catalogs/${version}
  # Fetch all regions for AWS
  # Note: SkyPilot already uses all global regions for AWS, and will periodically update the price

>   # manually running this command will disable the automatic update.

Reviewer: ```suggestion
  # Manually running this command will disable the automatic update.
```

>   # python -m sky.clouds.service_catalog.data_fetchers.fetch_aws --all-regions

Reviewer: Do we want to leave this out? Or, say some reason why users may want to run this?

Author: Good point! Removed it for now.

.. code-block:: bash

  version=$(python -c 'import sky; print(sky.clouds.service_catalog.constants.CATALOG_SCHEMA_VERSION)')
  mkdir -p ~/.sky/catalogs/${version}
  cd ~/.sky/catalogs/${version}
  # Fetch all regions for AWS

>   # Note: By default, SkyPilot already uses all global regions for AWS, and will periodically update the price

Reviewer: Hmm, this seems to conflict with L82?

Author: modified it a bit. How does that look?
